### PR TITLE
cargo-leptos: init at 0.1.8

### PR DESCRIPTION
**Pull request by benwis**
###### Description of changes
cargo-leptos is a build tool to allow easier generation of server side rendered sites for the Leptos web framework. While there is currently no website, they can be found on git [here](https://github.com/leptos-rs/leptos) and cargo-leptos [here](https://github.com/leptos-rs/cargo-leptos)
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [?] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x]  Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
  - [x] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Let me know if I've done something wrong, this is my first package!
This is exactly the same as #215400 but I fixed the merge conflict by rebasing against a more recent file. I would have used the old PR but it got closed as soon as I updated my fork

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1845,6 +1845,11 @@
     githubId = 75972;
     name = "Ben Booth";
   };
+  benwis = {
+    name = "Ben Wishovich";
+    email = "ben@benw.is";
+    github = "benwis";
+    githubId = 6953353;
```
Comment (alyssais) [inline maintainers/maintainer-list.nix:1852]: You're missing a `}`.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -15974,6 +15974,8 @@ with pkgs;
   cargo-edit = callPackage ../development/tools/rust/cargo-edit {
     inherit (darwin.apple_sdk.frameworks) Security;
   };
+  cargo-leptos = callPackage ../development/tools/rust/cargo-leptos {
+    inherit (darwin.apple_sdk.frameworks) Security;
```
Comment (alyssais) [inline pkgs/top-level/all-packages.nix:15978]: And here.

**Review by alyssais — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (benwis) [reply maintainers/maintainer-list.nix:1852]: I swear I can copy paste. Corrected

**Comment by benwis** [comment]
> how did a synapse commit get into here? 🫣

No idea lol

**Comment by RaitoBezarius** [comment]
> > how did a synapse commit get into here? 🫣
> 
> No idea lol. @Ma27 Do I need that change to synapse?

You included this commit through a "too broad" rebase probably.

**Comment by GabrielDougherty** [comment]
Result of `nixpkgs-review pr 222083` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cargo-leptos</li>
  </ul>
</details>

**Comment by Ma27** [comment]
It's in master here: https://github.com/NixOS/nixpkgs/commit/8c118951d586847f6c996ce718737b79017aa330

I guess the commit on your branch has a slightly different hash because of some rebase error and thus appears as difference. Long story short, removing the commit from your branch with another rebase should be sufficient :)

**Pull request closed without merging**